sample-app/src/pages/UniversalSearchPage.tsx

```diff
@@ -0,0 +1,29 @@
+import UniversalResults, { VerticalConfig } from '../components/UniversalResults';
+import { useLayoutEffect } from 'react';
+import { useAnswersActions } from '@yext/answers-headless-react';
+import '../sass/UniversalSearchPage.scss';
+
+const universalResultsFilterConfig = {
+  show: true
+};
+
+export default function UniversalSearchPage(props: { universalResultsConfig: Record<string, VerticalConfig>}) {
```

Comment:
nit, I think maybe we should make an interface for the props, and have it include universalResultsFilterConfig in it as well, seems a little odd that one is passed in and the other is hardcoded in this file?